Graphics/Implicit/Export/SymbolicFormats.hs

```diff
@@ -112,7 +112,7 @@ buildShared (WithRounding r obj) | r == 0 = build obj
 
 buildShared(UnionR _ _) = error "cannot provide roundness when exporting openscad; unsupported in target format."
 buildShared(IntersectR _ _) = error "cannot provide roundness when exporting openscad; unsupported in target format."
-buildShared(DifferenceR _ _ _) = error "cannot provide roundness when exporting openscad; unsupported in target format."
+buildShared(DifferenceR {}) = error "cannot provide roundness when exporting openscad; unsupported in target format."
```

Comment:
why quash the argument count just here?

Reply:
Hlint complains about it when there are 3 and more